README.md

```diff
@@ -1,2 +1,65 @@
 # htseq2multx
 
+Wrapper to [fastq-multx](https://github.com/brwnj/fastq-multx) to have it mimmic [barcode_splitter](https://bitbucket.org/princeton_genomics/barcode_splitter) and thus easily integrate with Princeton's [HTSEQ](http://htseq.princeton.edu) web interface.
+
+## Description
+
+The Princeton [HTSEQ](http://htseq.princeton.edu) software provides access to data generated by the [Genomics Core Facility](https://lsi.princeton.edu/facilities/sequencing-facility) in the [Lewis Sigler Institute for Integrative Genomics](https://lsi.princeton.edu).  It provides functionality for upstream analyses and data download/transfer.
+
+Part of the supported functionality of [HTSEQ](http://htseq.princeton.edu) is the demultiplexing of pooled sequencing samples.  The initial implementation of this functionality utilized an in-house python tool called [barcode_splitter](https://bitbucket.org/princeton_genomics/barcode_splitter/) written in the interpreted scripting language Python.  The performance of barcode_splitter appears to be I/O bound, so as sample capacity grows and costs fall, it has proven to not scale well.
+
+[fastq-multx](https://github.com/brwnj/fastq-multx) is a demultiplexing tool written in C and runs several orders of magnitude faster than [barcode_splitter](https://bitbucket.org/princeton_genomics/barcode_splitter).  Since [HTSEQ](http://htseq.princeton.edu) is highly integrated with [barcode_splitter](https://bitbucket.org/princeton_genomics/barcode_splitter), creating a wrapper to make [fastq-multx](https://github.com/brwnj/fastq-multx) look and act like [barcode_splitter](https://bitbucket.org/princeton_genomics/barcode_splitter) requires much less effort than altering [HTSEQ](http://htseq.princeton.edu) to use [fastq-multx](https://github.com/brwnj/fastq-multx).
+
+## Install
+
+This repo contains the details necessary to install the htseq2multx's module dependencies and script to both run the [fastq-multx](https://github.com/brwnj/fastq-multx) and convert its output.  It does not install [fastq-multx](https://github.com/brwnj/fastq-multx).  Refer to that [repo](https://github.com/brwnj/fastq-multx) for its installation instructions.
+
+### System-wide install
+
+    git clone https://github.com/hepcat72/htseq2multx.git
+    cd htseq2multx
+    perl Makefile.PL
+    make
+    sudo make install
+
+### Local/user-account install
+
+    git clone https://github.com/hepcat72/split-seq-processing.git
+    cd split-seq-processing
+    perl Makefile.PL INSTALL_BASE=~
+    make
+    make install
+
+Where `~` in the perl command causes the module to go into ~/lib.  Replace `~` with wherever you wish to install.
+
+### Caveats:
+
+Module dependencies should install automatically, but if any do not and `perl Makefile.PL` issues a warning such as:
+
+    Warning: prerequisite Readonly 2.05 not found.
+
+you can try, for example:
+
+    FTP_PASSIVE=1 PERL_MM_USE_DEFAULT=1 perl -MCPAN -e shell
+    cpan> install Readonly
```

Comment:
GTK.  Should I change the README?

Reply:
I think some update is warranted. Not sure how this should work exactly, but what I did worked fine assuming the user has cpanminus installed.

Reply:
Right.  I was concerned about that as well.  I'll consult my other packages.  I thought things were supposed to install automatically, but Readonly didn't for me, so perhaps I'm mistaken.  I'm pretty sure I've worked through this before in another distro.  I'll look.

- [ ] Figure out automatic module dependency installation

Reply:
OK.  Automatic installation is only for the modules included in lib/ in the distro.  The MakeMaker only checks the versions of already installed modules otherwise and reports on missing ones.  Looks like my other distro that handles automatic module installs uses a shell script that installs cpan minus (if not present) and installs dependencies that way.  There are apparently numerous ways to do it.  I'm going to read a bit more and decide which way to go.

Reply:
BTW, jury duty starts at 12:30 today.  I might not get this finished today due to that.